src/main/java/pl/futurecollars/invoicing/db/DatabaseConfiguration.java

```diff
@@ -48,4 +49,11 @@ public IdService idService(
         Path filePath = Files.createTempFile(databaseDirectory, idFile);
         return new IdService(filePath, filesService);
     }
+
+    @Bean
+    public static PropertySourcesPlaceholderConfigurer placeholderConfigurer() {
+        PropertySourcesPlaceholderConfigurer c = new PropertySourcesPlaceholderConfigurer();
+        c.setIgnoreUnresolvablePlaceholders(true);
+        return c;
+    }
```

Comment:
to jest tylko ignorowanie słusznych błędów - zamiast tego powinniśmy naprawić przyczynę. Błędami sypie dlatego, że public IdService idService(...) jest beanem tworzonym zawsze (brak conditionalOnProperty), a propertisy w @Value są skonfigurowane tylko w jednym profilu (prod) - ograniczenie tworzenia tego beana tylko do tego profilu pomoże